test/test_torch.py

```diff
@@ -13367,6 +13367,35 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    @onlyCUDA
+    def test_atomic_add(self, device):
+        # todo: we should probably expose a dtype.is_signed()
```

Comment:
Should file an issue for TODOs so we don't forget about them and link the issue from the TODO